portals/knowledge-management/src/Services/DetermineIfUserIsAuthenticated.js

```diff
@@ -45,7 +45,6 @@ async function determineIfUserIsAuthenticated(endpoint) {
         })
         .then((response) => {
             const isAuthenticated = response.status === 200;
-
```

Comment:
What is the purpose of removing this newline here?

It is prefered to have a good amount of whitespace for readability.

Reply:
By mistake space was removed. sorry for that.